eth/main.cpp

```diff
@@ -327,7 +327,7 @@ int main(int argc, char** argv)
 	std::string rpcCorsDomain = "";
 
 	string jsonAdmin;
-	ChainParams chainParams(genesisInfo(eth::Network::MainNetwork), genesisStateRoot(eth::Network::MainNetwork));
+	ChainParams chainParams;
```

Comment:
This doesn't seem right. Which chain params & genesis now `eth` without options starts with? Starting `eth` should have the same effect as `eth --mainnet`

Reply:
Ah, then the default (with no `--config` and no `--mainnet`) should be handled _after_ the `--mainnet` and `--testnet` cases https://github.com/ethereum/cpp-ethereum/blob/908d9f5b89d3fdbf421522d68bd03130ed6f4ba8/eth/main.cpp#L598-L601.  Otherwise the mainnet genesis accounts are needlessly loaded (and seem to interfere with the `--config` params).